CHANGELOG.md

```diff
@@ -38,6 +45,8 @@ in the naming of release branches.
 
 ### Changed
 
+- Changed UtxoEnv by replacing pool info (Map (KeyHash 'StakePool c) (PoolParams c)) with DPState
+    to show differences in two NewEpochState's. G
```

Comment:
we need to also mention the change in the API function `evaluateTransactionBalance`.

Reply:
done

Reply:
Also there is some lonely `G` at the end of this entry